ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -91,12 +91,16 @@ const SnapAuthorshipHeader = ({
           display={Display.Flex}
           justifyContent={JustifyContent.center}
           alignItems={AlignItems.center}
+          style={{
+            maxWidth: '236px',
+          }}
```

Comment:
What if the screen width changes ?

Reply:
My assumption was that this would be only popup with fixed width. But, you're right, I changed approach just in case.

Here are the results:
![Screenshot 2024-10-10 at 12 24 09](https://github.com/user-attachments/assets/9461060b-d06d-41d7-9471-bd43d5e92891)
![Screenshot 2024-10-10 at 12 24 26](https://github.com/user-attachments/assets/1cfd9626-2fc9-4cb7-ad7a-761af7927796)
![Screenshot 2024-10-10 at 12 24 40](https://github.com/user-attachments/assets/f4160341-79c6-438b-ab33-d1b17cd43fcd)